plugins/janus_audiobridge.c

```diff
@@ -7756,7 +7769,8 @@ static void *janus_audiobridge_mixer_thread(void *data) {
 							buffer[i] += (resampled[i]*p->volume_gain)/100;
 						}
 					}
-				} else {
+				}
+				if(groups_num != 0) {
```

Comment:
Why isn't this `} else if(groups_num != 0) {`? And why doesn't this have the to check as well? Not sure why exclude isn't part of this check.

Reply:
> Why isn't this } else if(groups_num != 0) {?
Just was thinking it's a little bit more clear... will fix.

> And why doesn't this have the to check as well? 
`to` handling a little bit later by add samples directly to buffers for specific user.
similar with `exclude` - it handling a little bit later by subtracting samples from buffers for specific user

P.S. I think there is another issue is there - I should take into account group notification - will fix...